msg/RadarTrack.msg

```diff
@@ -0,0 +1,12 @@
+# All variables below are relative to the radar's frame of reference.
+# This message is not meant to be used alone but as part of a stamped or array message.
+
+uint16 track_id                             # The track ID of the object generated by the radar.
+
+                                            # NB: The z component of these fields is ignored for 2D tracking.
+geometry_msgs/Point position                # x, y, z coordinates of the centroid of the object being tracked.
+geometry_msgs/Vector3 velocity              # The velocity of the object in each spacial dimension.
```

Comment:
spatial is the more common spelling: https://www.merriam-webster.com/dictionary/spatial